.github/workflows/tests.yml

```diff
@@ -6,21 +6,23 @@ on:
   pull_request:
 
 jobs:
-  core-tests:
-    runs-on: ubuntu-18.04
+  python-tests:
+    name: Python tests
+    runs-on: ubuntu-latest
 
     steps:
       - name: Cancel Previous Runs
         uses: styfle/cancel-workflow-action@0.4.1
         with:
           access_token: ${{ github.token }}
 
-      - uses: actions/checkout@v2
+      - name: Checkout The Walrus
+        uses: actions/checkout@v2
 
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.6
+          python-version: 3.7
```

Comment:
I'd probably set it to 3.8 or even 3.9 since I have a preference for newer versions to work, and just to not need to update it for a while. Although, I think it's fine to keep it at 3.7 as well.